release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3347,6 +3347,80 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the DNS-based egress firewall incorrectly prevented the creation of a firewall rule that contained a DNS name in uppercase characters. With this release, a fix to the egress firewall no longer prevents the creation of a firewall rule that contains a DNS name in uppercase characters. (link:https://issues.redhat.com/browse/OCPBUGS-51074[*OCPBUGS-51074*])
+
+* Previously, the control plane Operator did not honor the set `_PROXY` environment variables when it checked the API endpoint availability. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-50993[*OCPBUGS-50993*])
+
+* Previously, the availability set fault domain count was hardcoded to `2`. This value works in most regions on {azure-first} because the fault domain counts are typically at least `2`, but failed in the `centraluseuap` and `eastusstg` regions. With this release, the availability set fault domain count in a region is set dynamically so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-50966[*OCPBUGS-50966*])
+
+* Previously, when a pod ran on a node with the egress IPv6 assigned to it, the pod was unable to communicate with the Kubernetes service in a dual stack cluster. This resulted in dropped traffic with the IP family. With this release, only the source network address translation (SNAT) for the IP family that the egress IPs applied to is deleted, eliminating the risk of dropped traffic. (link:https://issues.redhat.com/browse/OCPBUGS-50594[*OCPBUGS-50594*])
+
+* Previously, a custom Security Context Constraint (SCC) impacted pods that the Cluster Version Operator generated from receiving a cluster version upgrade. With this release, {product-title} now sets a default SCC to each pod, so that any custom SCC created does not impact a pod. (link:https://issues.redhat.com/browse/OCPBUGS-50590[*OCPBUGS-50590*])
+
+* Previously, you could not install an cluster on {aws-short} in the `ap-southeast-5` region or other regions because the {product-title} internal registry did not support these regions. With this release, the internal registry is updated to include the following regions so that this issue no longer occurs:
++
+** `ap-southeast-5`
+** `ap-southeast-7`
+** `ca-west-1`
+** `il-central-1`
+** `mx-central-1`
++
+(link:https://issues.redhat.com/browse/OCPBUGS-49696[*OCPBUGS-49696*])
+
+* Previously, the Konnectivity proxy used by the `openshift-apiserver` in the control plane resolved registry names with cloud API suffixes on the control plane and then attempted to access them through the data plane.
```

Comment:
```suggestion
* Previously, a hosted cluster using the no-egress feature in ROSA and a container registry accessed through Amazon Virtual Private Cloud (VPC) endpoint failed to install because the `imagestreams` used by the container registry could not resolve. This was caused by the Konnectivity proxy using the `openshift-apiserver` in the control plane to resolve registry names with cloud API suffixes. With this release, the Konnectivity proxy resolves and routes hostnames consistently. (link:https://issues.redhat.com/browse/OCPBUGS-46466[*OCPBUGS-46466*])
```